connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -289,6 +290,23 @@ public void testBuildRestartPlanForUnknownConnector() {
         assertFalse(mayBeRestartPlan.isPresent());
     }
 
+    @Test
+    public void testConfigValidationNullConfig() {
```

Comment:
Could you also add a test case with multiple null values as well?

Reply:
yes, done